### ISSUE TITLE
Unset user permissions are added to the permissions

### ISSUE THREAD

**Issue by Al-Demon** [david-szabo97/grav-plugin-admin-addon-user-manager#38]
Please I don't know what I'm doing wrong...

I create a group call editor with this permissions:
```yaml
editor:
  groupname: editor
  readableName: editor
  description: writer
  access:
    admin:
      login: 'true'
      pages: 'true'
    admin-addon-user-manager: {  }
    site: {  }
```
And I have a new user:
```yaml
groups:
  - editor
access:
  admin:
    super: ''
    login: ''
    cache: ''
    configuration: ''
    configuration_system: ''
    configuration_site: ''
    configuration_media: ''
    configuration_info: ''
    settings: ''
    pages: ''
    maintenance: ''
    statistics: ''
    plugins: ''
    themes: ''
    users: ''
  admin-addon-user-manager:
    users: ''
    groups: ''
    users_expert: ''
  site:
    login: ''
```
_ _ _

And I can't login... only if I configured the user admin login true... but I could do this before without the plugin... I think this plugin is for create group, not for configure each user, right?

Thanks for all

**Comment by david-szabo97** [comment]
@Al-Demon Sorry, I don't really understand you.

You need to add `admin.login` permission to the group/user to let them login to the admin panel. Otherwise they can't login.

This plugin is a full-blown user and group editor.

**Comment by Al-Demon** [comment]
Ok, maybe I don't explain myself good... I put on group permission admin login enabled but I think the user who is inside the group don't inherit the permission and I have to configurate again admin login enabled and the rest of permission.

**Comment by david-szabo97** [comment]
You are overwriting the admin.login permission in your user file. Please
reinclude your group and user file in a code block.

**Comment by Al-Demon** [comment]
You right! If I change the password from the user manager overwriting user permissions... we need to be aware about this... Very thanks for all

**Comment by david-szabo97** [comment]
@Al-Demon 
I just checked why this happens and this is actually an unexpected behaviour.
When a specific permission is set to `Unset` it should not add it to the `access` array in the user config.

I will push a hotfix.

Thank you ;)

**Comment by david-szabo97** [comment]
Fixed in v2.1.3
